src/nanoarrow/meson.build

```diff
@@ -64,6 +64,25 @@ incdir = include_directories('..')
 nanoarrow_dep = declare_dependency(include_directories: [curdir, incdir],
                                    link_with: nanoarrow_lib)
 
+if get_option('ipc')
+    cmake = import('cmake')
+    cmake_opts = cmake.subproject_options()
+    cmake_opts.add_cmake_defines({'CMAKE_POSITION_INDEPENDENT_CODE': true})
+    flatcc_subproj = cmake.subproject('flatcc', options: cmake_opts)
+    flatcc_dep = flatcc_subproj.dependency('flatccrt')
```

Comment:
OK cool will take a look. I believe flatcc is trying to switch to Meson as well, so yea can look and see if that makes for a good upstream contribution

Reply:
I think this is solved in https://github.com/apache/arrow-nanoarrow/pull/483/files/82a6b48e4218f24b320cfb9e05f972c59d2cdd37..b878e23bb721f07a058e9f63a4db3aa6588f0faa 

Basically Meson allows you to provide your own "patch" file for systems that don't use Meson. The file(s) go in `subprojects/packagefiles/<patch_dir>` and the wrap system still takes care of fetching / downloading the library for you, rather than requiring you to vendor into your source tree

Reply:
Ugh set up the subproject here correctly but forgot to come back and update the code block here. Will push up a follow up PR to correct